sncast_scripts/Scarb.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "sncast_scripts"
+version = "0.17.0"
```

Comment:
```suggestion
version = "0.1.0"
```

Does this need to be pegged to the latest release from the library?

Reply:
Mmm no, probably not. Good call